doc/refcard/refcard.tex

```diff
@@ -288,18 +294,57 @@
   \columnbreak
 %% ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  \Sect{ESS developer}
+  \Sect{Package development}
 %% ~~~~~~~~~~~~~~~~
-  Evaluate your code into the package (in \textbf{\underline{ess-dev-map}}:
-  \textbf{C-c C-t}):
-
+  Evaluate your code into a given package (in \texttt{ess-dev-map},
+  \texttt{C-c C-t}):
+  
+  \begin{tabI}
+    \texttt{C-s, s} & Select package for namespaced evaluation\\
+  \end{tabI}
+  
+  Keybindings in \texttt{ess-r-package-mode} (\texttt{C-c C-w})\\
+  (call to \texttt{devtools} functions):
+  
   \begin{tabI}
-    \texttt{C-t, t} & Toggle developer on/off\\
-    \texttt{C-a, a} & Add package to the dev list\\
-    \texttt{C-r, r} & Remove package from dev list\\
+    \texttt{C-b, b} & Build package (\texttt{devtools::build()})\\
+    \texttt{C-c C-c} & Check package (\texttt{devtools::check()})\\
```

Comment:
There are multiple commands on C-c. Would be good to list all.

Reply:
Indeed, I was wondering which ones should be included. The help in ESS lists three commands (and five keybindings):
```
C-c C-w C-c C-c			ess-r-devtools-check-package
C-c C-w C-c C-w			ess-r-devtools-check-with-winbuilder
C-c C-w C-c c			ess-r-devtools-check-package
C-c C-w C-c h			ess-r-rhub-check-package
C-c C-w C-c w			ess-r-devtools-check-with-winbuilder
```
In fact, `devtools::build_win()` seems to be deprecated (and will be removed soon), so I didn't know if `C-c C-w` should be included here.
For `C-c h`, no problem.

Reply:
Hmm, I found it pretty useful. Any idea why they removed it? Is there a replacement?

Reply:
It seems that the old `build_win()` has been replaced by three distinct functions, `check_win_release()`, `check_win_devel()` and `check_win_oldrelease()`: https://github.com/r-lib/devtools/issues/1598
I suppose it might require some changes in ESS.

Reply:
Yes. Your changes are good, but we probably will wait a bit before moving to those functions in order to keep it working on older versions for a bit longer.